pkg/metrics/metrics.jsx

```diff
@@ -1595,6 +1610,7 @@ class MetricsHistory extends React.Component {
             selectedDate: null,
             packagekitExists: false,
             isBeibootBridge: false,
+            isPythonPCPInstalled: false,
```

Comment:
This smells like generating flicker. Let's initialize to `null`, and fix the comparison below.